ProblemH.cpp

```diff
@@ -2,8 +2,7 @@
 #include <cmath>
 using namespace std;
 
-int main()
-{
+int main() {
     float A, B;
```

Comment:
A challenge: Can you solve this problem without using `float`s nor `cmath` functions?

I.e., think of mathematical formulas that achieve floor, ceil, and round, using only integer math.

`A / B` already achieves flooring. Can you think of a formula for ceiling & rounding?

Hint:

```
10 / 5 = 2
11 / 5 = 2
12 / 5 = 2
13 / 5 = 2
14 / 5 = 2
15 / 5 = 3
16 / 5 = 3
17 / 5 = 3
18 / 5 = 3
19 / 5 = 3
```

Reply:
For why this could be useful: Because float operations are not exact. In a very bad case, the division result could be, e.g., `2.99999` instead of `3` and then its floor would 2 instead of 3.

(Tbh, not sure if such case exists. But a general rule of thumb is to use floats only when you can't use ints && you don't care about precision)